deploy/template.yaml

```diff
@@ -65,6 +65,7 @@ Mappings:
       assetsDomain: ""
       assetsPath: ""
       contactUrl: "https://home.build.account.gov.uk/contact-gov-uk-one-login"
+      templateCaching: false
```

Comment:
I think this can be true in all deployed environments - just not local (especially if you might be load testing against dev!)

I was actually thinking potentially the opposite - if you think there are any potential concerns about this, you could leave it off in staging & production. Should be safe enough if you've tested yourself...

Reply:
I was thinking if it's being worked on (and tested in dev envs) and nunjucks are getting edited regularly then it could be irritating - but is it being worked on locally rather than in the dev env? (A question for Hilux possibly)

I can make an edit for my dev, but I guess you're right that it would be more wanted if we're expecting people to perf test in dev more often